processes/ingest/chicago_isac.py

```diff
@@ -44,9 +51,9 @@ def processChicagoISACRecord(self, record):
             self.storePDFManifest(chicagoISACRec.record)
             self.addDCDWToUpdateList(chicagoISACRec)
             
-        except (MappingError, HTTPError, ConnectionError, IndexError, TypeError) as e:
+        except Exception as e:
             logger.exception(e)
-            logger.warn(ChicagoISACError('Unable to process ISAC record'))
+            logger.warning(ChicagoISACError('Unable to process ISAC record'))
```

Comment:
I think we can remove this line. We log the exception above.

Reply:
Agreed